.travis.yml

```diff
@@ -24,7 +24,9 @@ env:
     - COMPILER=/usr/bin/gcc
 
 before_script:
-  - docker pull xaptum/enftun:0.1.0
+  - pushd ${TRAVIS_BUILD_DIR}/.docker
```

Comment:
Oooh, we don't even have to push to Docker Hub. It will build the image each time.  Nice!